docs/source/cc_service.md

```diff
@@ -0,0 +1,297 @@
+# Chaincode as an external service
+
+Fabric v2.0 supports chaincode deployment and execution outside of Fabric that enables users to manage a chaincode runtime independently of the peer. This facilitates deployment of chaincode on Fabric cloud deployments such as Kubernetes. Instead of installing the chaincode on every peer, chaincode can now run as a service as opposed to a Fabric launched client. This capability leverages the Fabric v2.0 external builder and launcher functionality which enables operators to extend a peer with programs to build, launch, and discover chaincode. Before reading this topic you should become familiar with the [External Builds and Launchers](./cc_launcher.html) content.
```

Comment:
> Instead of installing the chaincode on every peer, chaincode can now run as a service as opposed to a Fabric launched client.

Grammar problem here.  

Instead of building and launching the chaincode on every peer, chaincode can now run as a service who's lifecycle is managed outside of Fabric.

Reply:
done